pdcupdater/handlers/retirement.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
@mprahl Sure thing! I filed https://github.com/fedora-infra/pdc-updater/issues/65 about it.